pkgs/applications/science/math/readstat/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, gettext, libtool }:
+
+stdenv.mkDerivation rec {
+  name = "readstat";
+  version = "1.1.8";
+
+  src = fetchFromGitHub {
+    owner = "WizardMac";
+    repo = "ReadStat";
+    rev = "v${version}";
+    sha256 = "1r04lq45h1yn34v1mgfiqjfzyaqv4axqlby0nkandamcsqyhc7y4";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  buildInputs = [ libtool ];
```

Comment:
You are sure about this? libtool normally goes to nativeBuildInputs and should be included in autoreconfHook.

Reply:
I barely write C/C++...  But that makes sense.  I'm assuming gettext goes in nativeBuildInputs as well then?